packages/frontend-app/src/app/components/Toast/Toast.stories.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from "react";
+import { Toast } from "./Toast";
+
+import type { Meta, StoryObj } from "@storybook/react";
+import { Button } from "../Button/Button";
+
+const meta: Meta<typeof Toast> = {
+  title: "Toast",
+  component: Toast,
+};
+
+export default meta;
+type Story = StoryObj<typeof Toast>;
+
+const ButtonWithHooks = () => {
```

Comment:
Why do we have this called "ButtonWithHooks" 😮‍💨 .

Reply:
Named it ButtonWithHooks because of the presence of useState and in storybook documentation, they have a name ButtonWithHooks and it's easily understandable @gbenga504